src/test/java/org/apache/sling/distribution/agent/impl/PrivilegeDistributionRequestAuthorizationStrategyTest.java

```diff
@@ -131,23 +175,53 @@ public void testNoPermissionOnDelete() throws Exception {
     }
 
     @Test
-    public void testPermissionOnDelete() throws Exception {
+    public void testDefaultPermissionOnDelete() throws Exception {
         String jcrPrivilege = "somePermission";
-        PrivilegeDistributionRequestAuthorizationStrategy strategy = new PrivilegeDistributionRequestAuthorizationStrategy(jcrPrivilege);
+        String DEFAULT_DELETE_PRIVELEGE = "jcr:removeNode";
```

Comment:
Do we really need this? Can't we just use Privilege.JCR_REMOVE_NODE?
If we can't, please fix the typo (misspelt "privilege")

Reply:
Using Privilege.JCR_REMOVE_NODE will give it an expanded form which will not look good on the config manager UI and might confuse the admin that this is the way the privileges are supposed to be configured.

I will take care of the typo.